repository.py

```diff
@@ -1,15 +1,22 @@
 import pymongo
 from pymongo import MongoClient
 import json
+import os
 
 def get_connection():
     return MongoClient('localhost', 27017)['gitnomic']
-  
+
 def get_config():
+  if os.environ.get('LOCAL_DEV'):
+    return {
```

Comment:
what if the FRD instance ran in docker too and this was always just pulled from the environment config, so the environment config decides how the application behaves differently in different environments rather than the code having to decide that.